docs/abi.md

```diff
@@ -51,15 +51,16 @@ Two specific sets of integer values are used in multiple places in the ABI:
 
 ## Exported Function
 
-Each Oak Module must expose the following **exported function** as a
-[WebAssembly export](https://webassembly.github.io/spec/core/syntax/modules.html#exports):
-
-- `oak_main: (u64) -> u32`: Invoked when the Oak Manager executes the Oak Node;
-  a handle for the read half of an initial channel is passed in as a parameter.
-
-The `oak_main` function for each Node should perform its own event loop, reading
+Each Oak Module must expose a single **exported function** as a
+[WebAssembly export](https://webassembly.github.io/spec/core/syntax/modules.html#exports),
+with signature `(u64) -> u32`. This function is invoked when the Oak Manager
+executes the Oak Node; a handle for the read half of an initial channel is
+passed in as a parameter. The name of this "oak_main" function for a Node is
```

Comment:
I would avoid referring to it as "oak_main", how about we keep calling it "entry point" (but without quotes)?

So this would read like

> The name of this entry point function for a Node is provided as part of ...

and below

> The entry point function for each Node should perform ...

Reply:
Done.